Meta/build-root-filesystem.sh

```diff
@@ -50,10 +50,12 @@ if [ "$SERENITY_TOOLCHAIN" = "Clang" ]; then
     mkdir -p mnt/usr/include/"$SERENITY_ARCH"-pc-serenity
     $CP --preserve=timestamps -r "$TOOLCHAIN_DIR"/include/c++ mnt/usr/include
     $CP --preserve=timestamps -r "$TOOLCHAIN_DIR"/include/"$SERENITY_ARCH"-pc-serenity/c++ mnt/usr/include/"$SERENITY_ARCH"-pc-serenity
-elif [ "$SERENITY_ARCH" != "aarch64" ]; then
+elif [ "$SERENITY_ARCH" = "x86_64" ]; then
     $CP --preserve=timestamps "$SERENITY_SOURCE_DIR"/Toolchain/Local/"$SERENITY_ARCH"/"$SERENITY_ARCH"-pc-serenity/lib/libgcc_s.so mnt/usr/lib
     $CP --preserve=timestamps "$SERENITY_SOURCE_DIR"/Toolchain/Local/"$SERENITY_ARCH"/"$SERENITY_ARCH"-pc-serenity/lib/libstdc++.a mnt/usr/lib
```

Comment:
Do we build this already? Should the conditional here just be `else` instead of `elif x64 elif aarch`? (If we don't have a libstdc++ yet, as-is is fine, but eventually I think we want just one blood for both Intel and aarch64, right?

Oh, i suppose on aarch we copy the .1 file, while on Intel we don't.

Let's keep it as is for now. Maybe it can be unified some more later on.

Reply:
Right, the libgcc_s.so vs libgcc_s.so.1 difference is important here.

Regarding the libstdc++.a, we do actually build that one, but I think it is only needed when using the GCC port on serenity itself (?), and we are far away from that anyway. And if it is only needed by the GCC port, then the GCC port should be responsible for installing it.

Reply:
Thinking about it a bit more, I went ahead and unified the two code paths for aarch64 and x86_64 by just copying everything from the `"$SERENITY_ARCH"-pc-serenity/lib/*` to `/usr/lib/`. This works for both x86_64 and aarch64.